docs/reference/data-streams/downsampling-ilm.asciidoc

```diff
@@ -0,0 +1,548 @@
+                                                "bytes": {
+                                                    "type": "long",
+                                                    "time_series_metric": "gauge"
+                                                },
+                                                "limit": {
+                                                    "properties": {
+                                                        "pct": {
+                                                            "type": "float"
+                                                        }
+                                                    }
+                                                },
+                                                "node": {
+                                                    "properties": {
+                                                        "pct": {
+                                                            "type": "float"
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        "workingset": {
+                                            "properties": {
+                                                "bytes": {
+                                                    "type": "long",
+                                                    "time_series_metric": "gauge"
+                                                }
+                                            }
+                                        }
+                                    }
+                                },
+                                "name": {
+                                    "type": "keyword"
+                                },
+                                "start_time": {
+                                    "type": "date"
+                                }
+                            }
+                        },
+                        "host": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        },
+                        "namespace": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        },
+                        "node": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        },
+                        "pod": {
+                            "type": "keyword",
+                            "time_series_dimension": true
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+----
+// TEST[continued]
+
+////
+[source,console]
+----
+DELETE _index_template/*
+----
+// TEST[continued]
+////
```

Comment:
Delete template is not needed.

Reply:
I tried removing this but the CI checks failed, so I'll leave it in unless you have objections.